plugins/inlinetoolbar/plugin.js

```diff
@@ -105,6 +104,40 @@
 				this.parts.close.remove();
 			};
 
+			/**
+			 * @inheritdoc CKEDITOR.ui.balloonPanel#show
+			 * @member CKEDITOR.ui.inlineToolbarView
+			 */
+			CKEDITOR.ui.inlineToolbarView.prototype.show = function() {
+				if ( this.rect.visible ) {
+					return;
+				}
+				var that = this,
```

Comment:
If you take a look onto our [`.on()` listener adding method](https://docs.ckeditor.com/ckeditor4/docs/#!/api/CKEDITOR.event-method-on) you'll note that it has `scopeObj` as a third argument.

Simply pass `this` as a `scopeObj` and you'll be able to continue to use `this` as a valid context within the function.